### ISSUE TITLE
Disable Rules

### ISSUE THREAD

**Issue by angela-choo** [dequelabs/axe-core#267]
I having problems disabling rules.  Is the below syntax correct? 

```
exports.config = {
    plugins: [{
        axe: ({
            rules:[{
                "aria-required-children": { "enabled": false },
                "color-contrast": { "enabled": false }
            }]
        }),
        package: 'protractor-accessibility-plugin'
    }],
...
```

**Comment by marcysutton** [comment]
`rules` takes an object, not an array. It should look like this:

```
rules: {
                "aria-required-children": { "enabled": false },
                "color-contrast": { "enabled": false }
            }
```

If it still doesn't work with the correct structure, it should be addressed in the Protractor-accessibility-plugin repo.

**Comment by dylanb** [comment]
@angela-choo also, I am not sure what that syntax is that you presented. You will need to call `axe.configure({...})` with the structure @marcysutton provided substituted into the `...` above.

**Comment by angela-choo** [comment]
Thank you for your response.

@dylanb I was originally trying to install the [protractor-accessibility-plugin](https://github.com/angular/protractor-accessibility-plugin) to enable aXe in my config file.  I am able to install the plugin but not customize rules.  Knowing this, would I still need to call `axe.configure({...})` in protractor.conf.js?

**Comment by dylanb** [comment]
@angela-choo can you submit the issue in that project and reference this issue?

**Comment by angela-choo** [comment]
Closing issue because this can be fixed via angular/protractor-accessibility-plugin.  Thanks for your help!
